src/README.md

```diff
@@ -239,5 +239,29 @@ juju config trove nova-keypair=trove-keypair
 ```
 
 
+## Troubleshooting
+
+Each database instance deployed has **2 ports** for the:
+- `tenant-net`
+- `trove-net`
+
+The **tenant-net port** has a security group (`trove_sg-uuid`) that allows the RabbitMQ traffic.
```

Comment:
The tenant network security group basically allows all egress traffic and also allows ingress traffic specific to the database created. The security group that allows RabbitMQ traffic is the other security group, the management security group.

Reply:
Updated